localflavor/ar/forms.py

```diff
@@ -137,3 +137,62 @@ def _format(self, cuit, check_digit=None):
             check_digit = cuit[-1]
             cuit = cuit[:-1]
         return '%s-%s-%s' % (cuit[:2], cuit[2:], check_digit)
+
+
+class ARCBUField(CharField):
+    """
+    This field validates a CBU (Clave Bancaria Uniforme). A CBU is a 22-digits long
+    number. The first 8 digits denote bank and branch number, plus a verifying digit.
+    The remaining 14 digits denote an account number, plus a verifying digit.
+
+    More info:
+    http://www.clientebancario.gov.ar/default.asp
```

Comment:
Can you also add the wikipedia entry? Thanks.
https://es.wikipedia.org/wiki/Clave_Bancaria_Uniforme